openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -131,6 +139,40 @@ def _create_from_course(course):
         )
 
     @staticmethod
+    def _load_from_module_store(course_id):
+        """
+        Load a CourseDescriptor, create a new CourseOverview from it, cache the
+        overview, and return it.
+
+        Arguments:
+            course_id (CourseKey): the ID of the course overview to be loaded.
+
+        Returns:
+            CourseOverview: overview of the requested course.
+
+        Raises:
+            - CourseOverview.DoesNotExist if the course specified by course_id
+                was not found.
+            - IOError if some other error occurs while trying to load the
+                course from the module store.
+        """
+        store = modulestore()
+        with store.bulk_operations(course_id):
+            course = store.get_course(course_id)
+            if isinstance(course, CourseDescriptor):
+                course_overview = CourseOverview._create_from_course(course)
+                course_overview.save()
+                return course_overview
+            elif course is not None:
+                raise IOError(
+                    "Error while loading course {} from the module store: {}",
+                    unicode(course_id),
+                    course.error_msg if isinstance(course, ErrorDescriptor) else unicode(course)
+                )
+            else:
+                raise CourseOverview.DoesNotExist()
+
+    @staticmethod
     def get_from_id(course_id):
```

Comment:
same question as above: why is this a `staticmethod` instead of a `classmethod`?